### PR TITLE
Add test case for virsh domiflist --inactive.

### PR DESCRIPTION
**Pull request by liuzzfnst**
virsh domiflist has a new option '--inactive',
so add a test case for this.

### CODE REVIEW

**Comment by liuzzfnst** [comment]
The result is running on libvirt-1.2.8-16.el7.x86_64
```
# ./run -k --no-downloads -t libvirt --tests virsh.domiflist
SETUP: PASS (0.40 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /root/community/autotest/client/tests/virt-test/logs/run-2015-06-09-23.20.19/debug.log
TESTS: 11
(1/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.domid: PASS (24.15 s)
(2/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.uuid: PASS (23.16 s)
(3/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.name: PASS (23.26 s)
(4/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.inactive_option: PASS (22.87 s)
(5/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.iface_model.virtio_iface: PASS (23.78 s)
(6/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.iface_model.e1000_iface: PASS (28.46 s)
(7/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_valid_option.iface_model.rtl8139_iface: PASS (23.94 s)
(8/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_invalid_option.none: PASS (1.66 s)
(9/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_invalid_option.invalid_domid: PASS (1.76 s)
(10/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_invalid_option.invalid_uuid: PASS (1.67 s)
(11/11) type_specific.io-github-autotest-libvirt.virsh.domiflist.with_invalid_option.addition_invalid_param: PASS (1.73 s)
TOTAL TIME: 176.51 s (02:56)
TESTS PASSED: 11
TESTS FAILED: 0
SUCCESS RATE: 100.00 %
```

File: libvirt/tests/src/virsh_cmd/domain/virsh_domiflist.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def add_iface(vm):
         """
         Attach interface for the vm
         """
-        if vm.is_alive():
+        if vm.is_alive() and "--inactive" not in additional_options:
```
Comment (PandaWei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domiflist.py]: Is there any relationship between "--inactive" and destroying VM?

Comment (liuzzfnst) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domiflist.py]: "--inactive" option shows the virtual interfaces that will be used on 
the next boot,
and we should add a virtual interface with 'virsh attach-interface 
--config' when
the domain is alive, it leads the attached interface can not be used 
until next boot.
Thus, we can see the difference between using "--inactive" option and 
not using.

File: libvirt/tests/src/virsh_cmd/domain/virsh_domiflist.py [hunk 2]
```diff
@@ -143,12 +144,14 @@ def add_iface(vm):
...
+        result = virsh.domiflist(options, additional_options,
+                                 ignore_status=True)
 
         if status_error == "yes":
             if result.exit_status == 0:
-                raise error.TestFail("Run passed for incorrect command \nCommand: "
-                                     "virsh domiflist %s\nOutput Status:%s\n"
+                raise error.TestFail("Run passed for incorrect command \n"
+                                     "Command: virsh domiflist %s\n"
+                                     "Output Status:%s\n"
```
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domiflist.py]: Is this update necessary?

**Comment by Chenditang** [comment]
@will-Do 
Updated, Thanks~~

**Comment by will-Do** [comment]
OK, I'm fine with PR

**Comment by waynesun09** [comment]
ACK and merging now, thx

**Pull request merged**